### PR TITLE
Fix coding standards

### PR DESCRIPTION
**Pull request by PedroTroller**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 language: php
 php:
-    - 5.3
```
Comment (Shivoham) [inline .travis.yml]: Please let the world use DictionaryBundle with php 5.3 :crying_cat_face: 
(yes, the whole world is using this bundle)

File: src/Knp/DictionaryBundle/Dictionary/DictionaryRegistry.php [hunk 2]
```diff
@@ -46,6 +71,12 @@ public function offsetGet($offset)
         return $this->dictionaries[$offset];
     }
 
+    /**
+     * @param mixed $offset
+     * @param mixed $value
+     *
+     * @throw \RuntimeException
```
Comment (akovalyov) [inline src/Knp/DictionaryBundle/Dictionary/DictionaryRegistry.php]: http://www.phpdoc.org/docs/latest/references/phpdoc/tags/throws.html

File: src/Knp/DictionaryBundle/Dictionary/DictionaryRegistry.php [hunk 3]
```diff
@@ -54,6 +85,11 @@ public function offsetSet($offset, $value)
         );
     }
 
+    /**
+     * @param mixed $offset
+     *
+     * @throw \RuntimeException
```
Comment (akovalyov) [inline src/Knp/DictionaryBundle/Dictionary/DictionaryRegistry.php]: http://www.phpdoc.org/docs/latest/references/phpdoc/tags/throws.html

File: src/Knp/DictionaryBundle/DependencyInjection/Compiler/DictionaryBuildingPass.php [hunk 4]
```diff
@@ -44,6 +54,12 @@ private function createDefinition($class, $name, array $dictionary)
         ;
     }
 
+    /**
+     * @param array $dictionary
+     *
+     * @return array
+     * @throw \RuntimeException
```
Comment (akovalyov) [inline src/Knp/DictionaryBundle/DependencyInjection/Compiler/DictionaryBuildingPass.php]: http://www.phpdoc.org/docs/latest/references/phpdoc/tags/throws.html

File: src/Knp/DictionaryBundle/Dictionary/DictionaryRegistry.php [hunk 5]
```diff
@@ -22,16 +31,32 @@ public function set($key, Dictionary $dictionary)
...
     public function offsetExists($offset)
     {
         return isset($this->dictionaries[$offset]);
     }
 
+    /**
+     * @param mixed $offset
+     *
+     * @return Dictionary
+     * @throw DictionaryNotFoundException
```
Comment (akovalyov) [inline src/Knp/DictionaryBundle/Dictionary/DictionaryRegistry.php]: http://www.phpdoc.org/docs/latest/references/phpdoc/tags/throws.html :)

**Pull request merged**